### DISCUSSION TITLE
Quarto files are not previewed correctly when treated as Markdown files

### DISCUSSION THREAD

**Discussion by rogerbramon** [community/community#38548] [Other Feature Feedback, Questions, & Ideas]
Since Quarto files are very similar to Markdown, I tried to set up linguist so that `.qmd` files are treated as Markdown files in Github.com and get nicely previewed. However, they are not correctly previewed, YAML metadata is shown as a table, but the content is shown as plain text and break lines are removed:

<img width="865" alt="image" src="https://user-images.githubusercontent.com/330441/200541624-255114a4-cfc5-4b21-a526-373c200c765e.png">

[File reference](https://github.com/rogerbramon/test-quarto/blob/main/linguist.qmd)

This is how I configured `.gitattributes`

```
*.qmd linguist-language=Markdown
```
I prepared a repo to illustrate the problem: [test-quarto](https://github.com/rogerbramon/test-quarto)

Do I miss any configuration? 

Thanks!

**Comment by ossanna16** [comment]
Hi there @rogerbramon and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by rogerbramon** [comment]
We would appreciate if anyone from the engineering team could give insight on this issue.

Thanks!
